backend/middlewares/sessionMiddleware.js

```
@@ -7,6 +1,13 @@
+const sessionMiddleware = (req, res, next) => {
+    if (req.session.user && req.session.user.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)